scripts/scripts.js

```diff
@@ -231,11 +263,34 @@ async function loadPage() {
     if (lastSection) lastSection.insertAdjacentElement('beforeend', a);
   }
 
-  setConfig({ ...CONFIG, miloLibs: LIBS });
+  let eventConfigItems;
+
+  if (eventUtils) {
+    eventConfigItems = {
+      decorateArea: () => eventUtils?.decorateEvent(document),
+      externalLibs: [
+        {
+          base: EVENT_LIBS,
+          blocks: eventUtils?.EVENT_BLOCKS, // or your custom EVENT_BLOCKS_OVERRIDE
```

Comment:
Don't need the optional chaining if you're in a conditional that checks for eventUtils